src/scripts/helpers.js

```diff
@@ -0,0 +1,54 @@
+export const getRandomCell = (gridSize) => Math.floor(Math.random() * gridSize);
```

Comment:
I think this can be more reabable
```suggestion
export const getRandomIndex = (gridSize) => Math.floor(Math.random() * gridSize);
```